### PR TITLE
Use 201H1 all the time

### PR DESCRIPTION
**Pull request by Tratcher**
This will get us much better coverage of newer windows features. The main question is if this queue can handle the load yet.

Contributes to https://github.com/dotnet/aspnetcore-internal/issues/3170.

### CODE REVIEW

**Comment by Tratcher** [comment]
@jkotalik @HaoK those are some interesting IIS setup failures. The tests are passed on this agent type in the helix matrix tests (unless they were skipped?). Any ideas?

**Comment by HaoK** [comment]
Hrm, are all those failures related to cert installation?

**Comment by Tratcher** [comment]
Those are the only failures I see, though there are a few at the top of the logs that indicate IIS Express isn't installed?

**Comment by HaoK** [comment]
Can we add more logging to the setup operations so it prints out what its trying to do?  Seems like a good opportunity to make this easier to debug for this, and the future...?

**Comment by jkotalik** [comment]
Hm interesting. So [this script](https://github.com/dotnet/aspnetcore/blob/master/src/Servers/IIS/tools/UpdateIISExpressCertificate.ps1) was added to run on our CI because of some issues with the default certs for IISExpress. This was added quite a long time ago (2 years+) so I don't recall the exact details. So we went through all certs for ports 443** and recreated the cert.

You can try removing this script and see if the tests pass?

**Comment by Tratcher** [comment]
Better, at least the IIS/Express tests aren't failing now. However, I don't think they're even running, they're set to skip if IIS/Express isn't installed.

The HttpSys failures are all HTTPS tests that try to re-use the IIS Express certificate bindings, which aren't present because of IIS Express not being installed.

**Comment by jkotalik** [comment]
> Better, at least the IIS/Express tests aren't failing now. However, I don't think they're even running, they're set to skip if IIS/Express isn't installed.

Oh, IIS Express isn't installed? That should be fixed. @dotnet/aspnet-build do you know who to contact about adding IISExpress to these new 201H1 agents?

**Comment by dougbu** [comment]
>  do you know who to contact about adding IISExpress to these new 201H1 agents?

@Tratcher has been working with the right Core-Eng team members on the 20H1 agents. Suggest commenting on that issue (wherever it is) or continuing that thread.

**Comment by Tratcher** [comment]
Better, the IIS Express and HttpSys tests pass now. But now the IIS tests are failing.
```
System.AggregateException : One or more errors occurred. (Operation did not succeed after 15 retries (Retrieving the COM class factory for component with CLSID {2B72133B-3F5B-4602-8952-803546CE3344} failed due to the following error: 80040154 Class not registered (0x80040154 (REGDB_E_CLASSNOTREG)).)
```

I know IIS isn't installed yet on this new queue, but I thought those tests were better about skipping if it wasn't installed?

File: eng/targets/Helix.Common.props [hunk 1]
```diff
@@ -21,7 +21,7 @@
   <!-- PR(ci.yaml) required queues -->
   <ItemGroup Condition="'$(IsRequiredCheck)' == 'true' AND '$(TargetArchitecture)' == 'x64'">
     <HelixAvailableTargetQueue Include="Ubuntu.1604.Amd64.Open" Platform="Linux" />
-    <HelixAvailableTargetQueue Include="Windows.10.Amd64.Open" Platform="Windows" />
+    <HelixAvailableTargetQueue Include="Windows.10.Amd64.Server20H1.Open" Platform="Windows" />
```
Comment (jkotalik) [inline eng/targets/Helix.Common.props:24]: How many machines are in this target queue? I'd be concerned if there isn't enough machines.

**Review by jkotalik — APPROVED** [review]

File: eng/targets/Helix.Common.props (hunk 1 above)
Comment (Tratcher) [reply eng/targets/Helix.Common.props:24]: We've already been using it for nightly runs. The more we use it the more machines they allocate to it. There may be a temporary lag while it catches up.

**Review by HaoK — APPROVED** [review]

**Pull request merged**